pkgs/development/libraries/wlroots/default.nix

```diff
@@ -23,9 +23,13 @@ stdenv.mkDerivation rec {
   buildInputs = [
     libGL wayland wayland-protocols libinput libxkbcommon pixman
     xcbutilwm libX11 libcap xcbutilimage xcbutilerrors mesa
-    libpng ffmpeg
+    libpng ffmpeg libuuid xorg.xcbutilrenderutil xwayland
```

Comment:
put `xcbutilrenderutil` explicitly as dependency (and use `inherit (xorg) xcbutilrenderutil;` in the callPackage).

Reply:
@AndersonTorres do we have a convention for that? I was a bit surprised about this but then again it shouldn't really matter anyway.